ui/src/helpers/utils.ts

```
@@ -48,3 +49,30 @@ export function tailwindClassNames(...args: ClassNamesArg[]) {
   return overrideTailwindClasses(classNames(args))
   // return classNames(args)
 }
+
+export function getPublicUrl(url: string = '') {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)